plugin/Cargo.toml

```diff
@@ -29,8 +29,8 @@ webpki = { git = "https://github.com/skyline-rs/webpki" }
 getrandom = { git = "https://github.com/skyline-rs/getrandom" }
 
 
-[patch.'https://github.com/skyline-rs/skyline-web']
-skyline-web = { git = "https://github.com/blu-dev/skyline-web", branch = "less-panics" }
```

Comment:
what is the reasoning for this change?

Reply:
> what is the reasoning for this change?

Blujay needs to update his fork to resolve build errors lol, which he won’t have time for for who knows how long. I assume this is used for the game modes page?